gonomics/runBenchmarks.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# uncomment below to install pysam
+# pip3 install --user pysam
+
+# uncomment below to install samtools (via homebrew)
+# brew install samtools
+
+for DIR in *BamRead; do
+	cd $DIR
+	go build
+	cd ..
+done
+
+tar -zxvf testdata.tar.gz
+
```

Comment:
Maybe this is a place where we could build the binaries that we currently include in the repo as binaries?

Reply:
Yes, that first loop is building the binaries. I included the binaries by accident.